Explorer/Assets/DCL/Multiplayer/Connections/Credentials/ConnectionStringCredentials.cs

```diff
@@ -0,0 +1,31 @@
+using DCL.Utilities.Extensions;
+using System;
+using System.Web;
+
+namespace DCL.Multiplayer.Connections.Credentials
+{
+    public readonly struct ConnectionStringCredentials : ICredentials
+    {
+        private readonly string connectionString;
+
+        public ConnectionStringCredentials(string connectionString)
+        {
+            this.connectionString = connectionString;
+        }
+
+        public string Url => connectionString
+                            .Split('?')
+                            .EnsureNotNull("Nothing to split via: {}")[0]!
+                            .Replace("livekit:", string.Empty);
+
+        public string AuthToken
+        {
+            get
+            {
+                var uri = new Uri(connectionString);
+                var query = HttpUtility.ParseQueryString(uri.Query);
+                return query["access_token"].EnsureNotNull("Access token not found");
```

Comment:
could it be also pre-processed once and than re-used? (If it used only once, then it doesn't matter of course)

Reply:
same as previous